lib/scenario_runner.py

```diff
@@ -176,9 +178,6 @@ def __init__(self,
 
         self._check_all_durations()
 
-        # we currently do not use this variable
```

Comment:
why remove that? I find comments like this which might indicate possible future implementations quite useful ...

Reply:
My thinking was, that having a variable `__filename` is not relevant anymore, because runner.py overrides the filename from the outside via `runner.set_filename(filename)` before each run. So a reset of __filename to the original filename (as one commented out line in the cleanup method indicates), would be counter-intuitive. 

Should I put the 3 commented out lines back into the code?